script/formatting/formatter.py

```diff
@@ -386,3 +413,19 @@ def check_file(file):
         if file.endswith(".ipynb"):
             notebook_file = os.path.join(EVA_NOTEBOOKS_DIR, file)
             check_notebook_format(notebook_file)
+
+    # CODESPELL
+    LOG.info("Codespell")
+    os.system("codespell eva/")
+    os.system("codespell docs/source/*/*.rst")
+    os.system("codespell docs/source/*.rst")
+    os.system("codespell *.md")
+    os.system("codespell eva/*.md")
+
+    # GO OVER ALL DOCS
+    LOG.info("aspell")
+    for elem in Path(EVA_DOCS_DIR).rglob('*.*'):
+        if elem.suffix == ".rst":
+            os.system(f"aspell --lang=en --personal=spelling.txt check {elem}")
```

Comment:
Breaking for errors in documentation makes sense.

Reply:
It looks like we need to run the command in a noninteractive way.

Reply:
There seems to be no easy way to run it in a noninteractive way.